src/apps/KintoAppRegistry.sol

```diff
@@ -46,6 +46,8 @@ contract KintoAppRegistry is
 
     uint256 public override appCount;
 
+    IKintoID public kintoID;
```

Comment:
if we make this immutable and add it tot he constructor we dont need the initialize v4